packages/ui/src/common/model/blocksToTime.ts

```diff
@@ -0,0 +1,16 @@
+import BN from 'bn.js'
+
+import { significantTimeUnit } from './significantTimeUnit'
+
+export function blocksToTime(blocks: BN) {
+  const seconds = blocks.toNumber() * 7
```

Comment:
There's a magic number and I think that it might be wrong :) (shouldn't it be 6 seconds?).

Please use a const loaded from one location (maybe the same as `1441--` payout periods?

Reply:
Payout periods are in Working Groups domain, while I believe this constant is more general - moved it to common/constants for now